db/migrate/20160226014344_create_pallets.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePallets < ActiveRecord::Migration
+  def change
+    create_table :pallets do |t|
+      t.references :cycle_count, index: true, foreign_key: true
+      t.string :pallet_number, null: false
+      t.boolean :pallet_in_grossman, default: false
+      t.boolean :pallet_in_location, default: false
```

Comment:
The idea of pallet_in_grossman would be for a tag that wasn't  in Grossman.  The scan should verify 3 things in my mind.
1) In Grossman and at the right location
2) In Grossman, but in a different location
3) Not in Grossman.